icu4c/source/common/unicode/rbbi.h

```diff
@@ -695,7 +700,7 @@ class U_COMMON_API RuleBasedBreakIterator /*U_FINAL*/ : public BreakIterator {
      * @param c         A character in the dictionary set
      * @internal (private)
      */
-    const LanguageBreakEngine *getLanguageBreakEngine(UChar32 c);
```

Comment:
This should not be changed, the return LanguageBreakEngine pointer have to be const. The returned LanguageBreakEngine should be immutable.

Reply:
Fixed